import org.apache.hadoop.fs.PathIOException;
public class FunctionUse14881 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
pathioexception.getPath();
}
}